django-backend/fecfiler/scha_transactions/views.py

```diff
@@ -29,6 +30,24 @@ class SchATransactionViewSet(CommitteeOwnedViewSet):
     ).all()
     """QuerySet: all schedule a transactions with an aditional contributor_name field"""
 
+    def get_queryset(self):
+        assert self.queryset is not None, (
+            "'%s' should either include a `queryset` attribute, "
+            "or override the `get_queryset()` method." % self.__class__.__name__
+        )
+
+        report_id = None
+        if self.request is not None:
+            report_id = self.request.query_params.get("report_id")
+
+        queryset = self.queryset
```

Comment:
since this class inherits `CommitteeOwnedViewSet`, use self.get_queryset() so we include the modifications it makes to the queryset.

In functionality terms, you need this to limit the results by committee